test/pylib/manager_client.py

```diff
@@ -123,29 +123,25 @@ async def servers(self) -> List[str]:
     async def mark_dirty(self) -> None:
         """Manually mark current cluster dirty.
            To be used when a server was modified outside of this API."""
-        await self._get("/cluster/mark-dirty")
+        await self._get_text("/cluster/mark-dirty")
```

Comment:
I mentioned this before. I read on an `aiohttp` issue not awaiting the text might cause problems.

Also, since the connection is local `AF_UNIX` it's practically free. This split `await` is for slow network connections so you can read headers before the whole request body finished.